<?xml version="1.0"?>

<launch>
  <node pkg="exposure_control" type="changeParam" name="changeParam" args="" output="screen" respawn="false" >
  </node>    
 <param name="step_length_aec" type="double" value=" 0.0000000000005" />
 <param name="step_length_gec" type="double" value="0.00000001" />
 <param name="upper_bound" type="double" value="50000" />
 <param name="target_id" type="int" value="8" />
 <param name="cam_name" type="string" value="/camera_array/cam0/image_raw" />
 <param name="info_mode" type="bool" value="true" />
 <!-- "aaec". "aec", "gec", "default" -->
 <param name="chosen_method" type="string" value="aaec" />
</launch>